plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -168,9 +169,8 @@ private static void generateMultipleAttributeSchema(JSONObject attributeSchema,
         } else {
             ConfigurationAsCode configurationAsCode = ConfigurationAsCode.get();
             try {
-
-                System.out.println("DocString: " + configurationAsCode.getHtmlHelp(Jenkins.getInstance().getDescriptor().getClass()
-                    , attribute.getName()));
+                System.out.println("DocString: " + configurationAsCode.getHtmlHelp(attribute.getGetter().getClass() ,
```

Comment:
@timja This too does not seem to return anything, but since I thought this should successfully describe the attribute.

Reply:
the getter isn't the descriptor so it's unlikely it would work